server/lib/models/account.go

```diff
@@ -88,30 +95,30 @@ func calc_sha256(source string) string {
 }
 
 //compute_sign is calculate signature of login request.
-func compute_sign(id, secret string, walltime uint64) string {
-	src := strconv.FormatUint(walltime, 10) + id + secret
+func compute_sign(id uint64, secret string, walltime uint64) string {
```

Comment:
__[gometalinter]__
* [golint] don't use underscores in Go names; func compute_sign should be computeSign